server/memory_store_test.go

```diff
@@ -49,21 +45,19 @@ func TestGetSet(t *testing.T) {
 	key, err := ms.Put(ctx, expected)
 	assert.NoError(t, err)
 
-	actual, err := ms.Get(ctx, key, common.BinaryDomain)
+	actual, err := ms.Get(ctx, key, BinaryDomain)
 	assert.NoError(t, err)
 	assert.Equal(t, actual, expected)
 }
 
 func TestExpiration(t *testing.T) {
-	t.Parallel()
-
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
 	kzgConfig := &kzg.KzgConfig{
-		G1Path:          "../e2e/resources/kzg/g1.point",
-		G2PowerOf2Path:  "../e2e/resources/kzg/g2.point.powerOf2",
-		CacheDir:        "../e2e/resources/kzg/SRSTables",
+		G1Path:          "../operator-setup/resources/g1_abbr.point",
```

Comment:
if we're gonna change from `e2e` to `operator-setup` then would it make sense to remove the SRS values stored in the `e2e` directory?

Reply:
Openning a PR to move enough points for 2MiB blobs to this repo, and removing the operator-setup submodule.